aopy/analysis.py

```diff
@@ -1010,3 +1011,144 @@ def linear_fit_analysis2D(xdata, ydata, weights=None, fit_intercept=True):
 
     return linear_fit, linear_fit_score, pcc_all[0], pcc_all[1], reg_fit
 
+######### Spectral Estimation and Analysis ############
+
+def get_sgram_multitaper( data, fs, win_t, step_t, nw=None, bw=None, adaptive=False):
+    """get_sgram_multitaper
+
+    Compute multitaper estimate from multichannel signal input.
+
+    Args:
+        data (nt, nch): nd array of input neural data (multichannel)
+        fs (int): sampling rate
+        win_t (float): spectrogram window length (in seconds)
+        step_t (_type_): step size between spectrogram windows (in seconds)
+        nw (float, optional): time-half-bandwidth product. Defaults to None.
+        bw (_type_, optional): spectrogram bandwidth. Defaults to None.
```

Comment:
there's still a couple of `_type_` values in here, should be `float`, probably? or is `bw` a tuple?